cpp/oneapi/dal/backend/interop/error_convertor.hpp

```diff
@@ -0,0 +1,271 @@
+/*******************************************************************************
+ * Copyright 2020 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+#include "daal/include/services/error_handling.h"
+#include "daal/include/services/internal/status_to_error_id.h"
+#include "oneapi/dal/exception.hpp"
+
+namespace oneapi::dal::backend::interop {
+
+void status_to_exception_default(daal::services::Status s);
+
+template <class local_convertor>
```

Comment:
Code style: `local_convertor` -> `StatusConverter`
Additionally, I'd recommend to rename it to `StatusConverter`.